extensions/sql-migration/src/dialog/skuRecommendationResults/skuRecommendationResultsDialog.ts

```diff
@@ -303,7 +303,10 @@ export class SkuRecommendationResultsDialog {
 			{ value: constants.SQL_TEMPDB },
 			{
 				value: recommendation.targetSku.tempDbDiskSizes!.length > 0
-					? constants.STORAGE_CONFIGURATION(recommendation.targetSku.logDiskSizes![0].size, recommendation.targetSku.logDiskSizes!.length)
+					? constants.STORAGE_CONFIGURATION(
```

Comment:
is it safe to assert that recommendation.targetSku.tempDbDiskSizes!.length exists?
wouldn't handling the null be a less fragile pattern?
recommendation.targetSku.tempDbDiskSizes!?length > 0

Reply:
taken the change.